app/components/document_title_component.rb

```diff
@@ -15,8 +15,20 @@ def object_type_class
   def version_banner
     return 'You are viewing the latest version.' if @presenter.head_user_version_view? || @presenter.current_version_view?
 
-    'You are viewing an older version.' if version_or_user_version_view?
+    "You are viewing an older #{version_type} version." if version_or_user_version_view?
```

Comment:
Is the `if` needed here?

Reply:
The `if` is here so that the function returns `nil` when we're on the latest version and there should not be a banner displaying. That's the `<% if version_banner %>`. Would it be better to move the check for whether a banner is needed into a separate method?

Reply:
No, fine as is.